vividus-plugin-azure-resource-manager/build.gradle

```diff
@@ -5,6 +5,8 @@ dependencies {
     implementation project(':vividus-extension-azure')
     implementation project(':vividus-soft-assert')
     implementation(group: 'com.azure.resourcemanager', name: 'azure-resourcemanager-resources', version: '2.40.0')
+    implementation(group: 'com.azure', name: 'azure-security-keyvault-secrets', version: '4.8.3')
```

Comment:
We can reuse approach from ResourceManagementSteps. I thought about moving requests to separate class and reusing in processor. But looks like it will be more complex, that builder from azure-security-keyvault-secrets library